code/datums/ai_laws.dm

```diff
@@ -209,26 +210,16 @@
 			continue
 
 		add_inherent_law(line)
-	if(!inherent.len) //Failsafe to prevent lawless AIs being created.
-		log_law("AI created with empty custom laws, laws set to Asimov. Please check silicon_laws.txt.")
-		add_inherent_law("You may not injure a human being or, through inaction, allow a human being to come to harm.")
-		add_inherent_law("You must obey orders given to you by human beings, except where such orders would conflict with the First Law.")
-		add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
-		WARNING("Invalid custom AI laws, check silicon_laws.txt")
-		return
 
 /* General ai_law functions */
 
 /datum/ai_laws/proc/set_laws_config()
 	var/list/law_ids = CONFIG_GET(keyed_list/random_laws)
```

Comment:
something about this call is causing runtime errors, the config option isn't entered into the entries list for some reason, causing an index error

Reply:
resolved? I think?